gcpy/benchmark.py

```diff
@@ -1747,14 +1756,16 @@ def make_benchmark_emis_tables(
     # ==================================================================
 
     # Emissions species dictionary
-    species = yaml.load(
+    spc_dict = yaml.load(
```

Comment:
Maybe better here to use `spc_dict = util.read_config_file(os.path.join(os.path.dirname(__file__), emission_spc.yml))`

Reply:
I'm still seeing several uses of `yaml.load` in this file:

```
158:     properties = yaml.load(open(properties_path), Loader=yaml.FullLoader)
427:     properties = yaml.load(open(properties_path), Loader=yaml.FullLoader)
1777:    spc_dict = yaml.load(
1782:    inv_dict = yaml.load(
2404:    newvars = yaml.load(
3385:    spcdb = yaml.load(open(path), Loader=yaml.FullLoader)
3395:    aod = yaml.load(open(path), Loader=yaml.FullLoader)
3826:    spc_properties = yaml.load(open(os.path.join(os.path.dirname(__file__),
4208:    properties = yaml.load(open(properties_path), Loader=yaml.FullLoader)
```

Would you recommend changing all of these? If so, maybe that's best done in a separate PR?

Reply:
Yeah maybe let's do a separate PR.

Reply:
OK, thanks. I just submitted a feature request so we don't forget. See https://github.com/geoschem/gcpy/issues/188.